modules/tinymce/src/core/test/ts/browser/dom/SerializerTest.ts

```diff
@@ -922,5 +922,65 @@ describe('browser.tinymce.core.dom.SerializerTest', () => {
       '',
       'Should remove br');
   });
+
+  it('Remove redundant br elements', () => {
```

Comment:
It seems that these are the same tests that is already in this file? Could you just go though them to double check that they are all the same. I'm guessing this was copied over when that deprecation PR was made or something but if we made changes or added tests to the DomParser tests since then we might be missing things.

- [ ] Remove duplicate tests

Reply:
they were all duplicated :sweat_smile: